@here/harp-mapview/lib/DataSource.ts

```diff
@@ -369,11 +502,19 @@ export abstract class DataSource extends THREE.EventDispatcher {
     getDisplayZoomLevel(zoomLevel: number): number {
         return THREE.MathUtils.clamp(
             zoomLevel + this.m_storageLevelOffset,
-            this.m_minZoomLevel,
-            this.m_maxZoomLevel
+            this.m_minDataLevel,
```

Comment:
Shouldn't this use the min/max display levels?

Reply:
No the display zoom level controls what level of a `DataSource` should be displayed(taking the storage offset into account and clamping it to the data level range). It doesn't control if the `DataSource` should be displayed at a given zoom level. Before this PR it was done by `canGetTile` on a per tile level. I introduced `isVisible(zoomLevel)` to get around this issue because it can be done for the whole `DataSource` instead, which potentially makes culling more efficient(no need to cull each tile if we know the whole `DataSource` shouldn't be displayed).

Reply:
> It doesn't control if the DataSource should be displayed at a given zoom level.

You are saying that `getDisplayZoomLevel` shouldn't dictate if the Datasource can be displayed at supplied zoom level? This to me is very confusing. This should be renamed probably to `getDataZoomLevel`, because if I understand you correctly it gets the data level that should be displayed at the supplied zoom level and not the display level.